src/main/java/teammates/ui/template/FeedbackTemplateQuestionInfo.java

```diff
@@ -0,0 +1,140 @@
+package teammates.ui.template;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+
+/**
+ * This class uses the information from feedbackSessionTeamEvaluationTemplate.json
```

Comment:
There is no need to mention `feedbackSessionTeamEvaluationTemplate.json`

See `StudentFeedbackResultsQuestionWithResponses.java` for example.